lesson5.html

```diff
@@ -0,0 +1,45 @@
+<html>
+    <head>
+        <title>Mouse leave and over event</title>
+        <link rel="stylesheet" type="text/css" href="style5.css">
+    </head>
+    <body>
+        <div id ="newBtn"></div>
+        <div class="btn-postn"><button id ="awsome">Just-Awsome</button></div>
+        
+        <script>
+            let awsmBttnCmp = document.getElementById("awsome");
+            let newCmp = document.getElementById("newBtn");
+            const label = "Im Awsome" ;
+             function onClick(event){
+                 console.log("Hey!! Be positive");
+             }
+             function mouseOverHandler (event){
+                 console.log("You are here");
+                 awsmBttnCmp.innerText = "Be Awsm" ;
+                 
+                 awsmBttnCmp.classList.add("button-shape");
+                 let bttnNew = document.createElement("button");
```

Comment:
Maybe move this line few lines below, so that first comes all the logic regarding awsmBttnCmp and then new button